core/db_client.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import json
+import pandas as pd
+
+
+class DBClient(object):
+
+	def __init__(self, config_filepath = 'database_config.json'):
+		"""
+		Set up DBClient with corresponding database credentials
+		"""
+		app = Flask(__name__)
+		with open(config_filepath) as f:
+			POSTGRES = json.load(f)
+		app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://%(user)s:%(pw)s@%(host)s:%(port)s/%(db)s' % POSTGRES
+		app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+		self.db = SQLAlchemy(app)
+		self.table_name = POSTGRES['table_name']
+
+	def get_labels(self):
+		"""
+		Get category_labels table
+		"""
+		labels = pd.read_sql_query("select * from {table_name}".format(table_name=self.table_name), self.db.engine)
+		return labels
+
+	def add_labels(self, data_providers, categories):
+		"""
+		Append new category labels to the category_labels table and replace old labels, where categories[i] corresponds 
+		to the labeled category for data_providers[i]
+		"""
+		labels = pd.read_sql_query("select * from {table_name}".format(table_name=self.table_name), self.db.engine)
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Use `self. get_labels()`.

Reply:
Also, what happens when getting the labels fails?

Reply:
Updated.

Getting the labels should only fail when the DB is down (and that should be a rare occurrence). Should I have error handling for that?